website/integrations/services/openproject/index.md

```diff
@@ -0,0 +1,91 @@
+---
+title: OpenProject
+---
+
+<span class="badge badge--secondary">Support level: Community</span>
+
+## What is OpenProject
+
+> OpenProject is a open source project management software.
```

Comment:
Ideally, there would be a few more sentences here, perhaps a use case, how people typically use this software.